packages/core/upload/server/services/image-manipulation.js

```diff
@@ -81,8 +81,13 @@ const optimize = async file => {
 
   const newFile = { ...file };
 
-  const { width, height, size, format } = await getMetadata(newFile);
+  try {
+    const { width, height, size, format } = await getMetadata(newFile);
```

Comment:
The `try ... catch` block should be extended to the code below, since these variables are being used there. Also, in case of error it needs to exit early, aside of printing error to the console.